doc/Language/quoting.pod6

```diff
@@ -174,38 +174,26 @@ My favorite color is blue!
 =end code
 
 X<|\ (quoting)>
+
 The C<qq> form – usually written using double quotes – allows for
-interpolation of backslash sequences and variables, i.e., variables can be
-written within the string so that the content of the variable is inserted into
-the string. It is also possible to escape variables within a C<qq>-quoted
-string:
+interpolation of backslash escape sequences (like C<q:backslash>), all sigiled
+variables (like C<q:scalar:array:hash:function>), and any code inside C<{...}>
+(like C<q:closure>).
 
-=begin code :allow<B> :preamble<my $color = 'blue'>
-say B<">The B<\>$color variable contains the value '$color'B<">;
-=end code
+=head3 Interpolating variables
```

Comment:
I'm not usually in favor of deletion, unless it's patently wrong and it's addressing some issue. That text is usually there for a reason, and it might be a subtle reason, but reasonable nonetheless.

Reply:
This code wasn't deleted, just moved.  In this commit, it's at lines 303–315.

Reply:
Fair enough. This is a subsection of a section that's called `Interpolation: qq`. Subsections create index entries, so you might want to flesh this out to something that more clearly shows the context or can be found more easily.